analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -101,10 +102,12 @@ class Unmanaged(
 
             else -> {
                 // For all non-GitRepo VCSes derive the name from the VCS URL.
+                val fileUrl = File(vcsInfo.url).takeIf { it.exists() } ?: File(URI(vcsInfo.url))
+
                 Identifier(
                     type = managerName,
                     namespace = "",
-                    name = vcsInfo.url.split('/').last().removeSuffix(".git"),
+                    name = fileUrl.name.removeSuffix(".git"),
```

Comment:
Have you considered to (also) change the code which puts the `\` into `VcsInfo.url`?
E.g. should that be changed to put "invariant separator paths" in there always?

Reply:
I'll take a look at that idea and eventually create a follow-up PR later.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/6407.